include/xtypes/idl/parser.hpp

```diff
@@ -1055,7 +1055,7 @@ class Parser
     {
         using namespace peg::udl;
         std::string name;
-        //std::map<std::string, Member> member_list;
+        //std::vector<Member> member_list;
```

Comment:
Is a TODO? Should be removed?

Reply:
It's a TODO, kept as a template for the future work (if uncommented, it raises an unused variable warning).